core/trino-main/src/test/java/io/trino/execution/TestSetTimeZoneTask.java

```diff
@@ -79,9 +79,23 @@ public void testSetTimeZoneLocal()
                 Optional.empty());
         executeSetTimeZone(setTimeZone, stateMachine);
 
+        assertThat(stateMachine.getSetSessionProperties()).hasSize(0);
+    }
+
+    @Test
+    public void testSetTimeZoneLocalAfterSetTimeZoneExecuted()
+    {
+        QueryStateMachine stateMachine = createQueryStateMachine("SET TIME ZONE 'America/Bahia_Banderas'");
```

Comment:
Not sure i follow. This `stateMachine` is worked on, but not asserted on.

Reply:
@findepi 

After checking, the test itself was meaningless because there was no connection between the first `QueryStateMachine` and the second `QueryStateMachine` object. 

When creating the second `QueryStateMachine`, do you think setting `setSessionProperties` of the first `QueryStateMachine` to the second `setSessionProperties` would be appropriate?